app/src/androidTest/java/com/duckduckgo/app/statistics/VariantManagerTest.kt

```diff
@@ -95,6 +94,23 @@ class VariantManagerTest {
         assertTrue(variant.hasFeature(SearchWidgetDaxCta))
     }
 
+    // Search Notification Experiment
+
+    @Test
+    fun searchNotificationControlVariantIsInactive() {
```

Comment:
This is the search notification, you want this to be active. Happy to zoom if you want to discuss this.